docs/scenarios.md

```diff
@@ -0,0 +1,601 @@
+
+```json
+// user settings
+{
+  "peacock.color": "#00ff00"
+}
+```
+
+```json
+// workspace settings
+{
+  "peacock.color": "#00ffff"
+}
+```
+
+- open a project
+- Peacock sees we are in local env., so it looks for workspace, then user settings for a "peacock.color".
+- Workspace settings has "peacock.color" #00ffff (yellow)
+- Peacock applies color customizations using #00ffff (yellow)
+- No user action was taken, so Peacock does not write the colors.
+
+```json
+// user settings
+{
+  "peacock.color": "#00ff00"
+}
+```
+
+```json
+// workspace settings
+{
+  "peacock.color": "#00ffff"
+}
+```
+
+- User resets all colors
+- Peacock removes all settings from workspace (color, remoteColor, colorCustomizations)
+
+```json
+// user settings
+{}
+```
+
+```json
+// workspace settings
+{}
+```
+
+### F - Remote / No User Settings / No Workspace Settings - [User Sets Remote Color]
+
+```json
+// user settings
+{}
+```
+
+```json
+// workspace settings
+{}
+```
+
+- open a project
+- Peacock sees we are in local env., so it looks for workspace, then user settings for a "peacock.color". None exist, so we do nothing.
+- User goes to remote environment
+- Peacock sees we are in remote env., so it looks for workspace, then user settings for a "peacock.remoteColor". None exist, so we do nothing.
+- User changes the color to #00ff00 (green)
+- Peacock applies color customizations using #00ff00 (green)
+- User changed the color so we want to write it.
+- Peacock checks if "peacock.remoteColor" in workspace has changed. It has. So "peacock.remoteColor" is written to workspace.
+
+```json
+// user settings
+{}
+```
+
+```json
+// workspace settings
+{
+  "peacock.remoteColor": "#00ff00"
+}
+```
+
+### G - Remote / No User Settings / Workspace Color - [User Sets Remote Color]
+
+```json
+// user settings
+{}
+```
+
+```json
+// workspace settings
+{
+  "peacock.remoteColor": "#00ff00"
+}
+```
+
+- open a project
+- Peacock sees we are in local env., so it looks for workspace, then user settings for a "peacock.color". None exist, so we do nothing.
+- User goes to remote environment
+- Peacock sees we are in remote env., so it looks for workspace, then user settings for a "peacock.remoteColor".
+- Local workspace has "peacock.remoteColor" #00ff00 (green)
+- Peacock applies color customizations using #00ff00 (green)
+- Peacock checks if "peacock.remoteColor" in workspace is green. It is. So "peacock.remoteColor" is not written.
+
+```json
+// user settings
+{}
+```
+
+```json
+// workspace settings
+{
+  "peacock.remoteColor": "#00ff00"
+}
+```
+
+- User changes the color to "#ffff00" (yellow)
+- Peacock applies color customizations using #ffff00 (yellow)
+- User changed the color so we want to write it.
+- Peacock checks if "peacock.remoteColor" in workspace is yellow. It is not. So "peacock.remoteColor" is written to workspace.
+
+```json
+// user settings
+{
+  "peacock.color": "#00ff00"
```

Comment:
user settings should still be {} at this point

Reply:
same typo. 

fixed.